src/main/java/hw_10/Application.java

```diff
@@ -0,0 +1,62 @@
+package hw_10;
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+
+public class Application {
+    public static void main(String[] args) {
+        LocalDateTime localDateTime = LocalDateTime.now();
+        System.out.println(localDateTime);
+
+        DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+        String date = localDateTime.format(dateTimeFormatter);
+        System.out.println("Date in format: " + date);
+
+        LocalDateTime dateTime = localDateTime.of(2025, 9, 3, 16, 50);
+        LocalDateTime Start = localDateTime.of(2025, 4, 15, 13, 20);
+        LocalDateTime end = localDateTime.of(2025, 4, 16, 16, 45);
+        int days = -5;
+        int hours = 8;
+        String pattern = "dd-MM-yyyy HH:mm";
+
+        calculateFutureDateTime(dateTime, days, hours);
+
+        if (isWeekend(dateTime)) {
+            System.out.println("true");
+        }
+        System.out.println("false");
```

Comment:
False will be printed anyway. So probably you need to use else if? And print something like "It is a working day"